doc/changelog.rst

```diff
@@ -6,6 +6,291 @@ Change Log
+  ODE integrator. The function `krylovsolve` is maintained for convenience
+  and now supports many more options.
+- The `sparse` parameter has been removed. Supply a sparse `Qobj` for the
+  Hamiltonian instead.
+
+Floquet Solver (fsesolve and fmmesolve):
+
+- The Floquet solver has been rewritten to use a new `FloquetBasis` class
+  which manages the transformations from lab to Floquet basis and back.
+- Many of the internal methods used by the old Floquet solvers have
+  been removed. The Floquet tensor may still be retried using
+  the function `floquet_tensor`.
+- The Floquet Markov Master Equation solver has had many changes and
+  new options added. The environment temperature may be specified using
+  `w_th`, and the result states are stored in the lab basis and optionally
+  in the Floquet basis using `store_floquet_state`.
+- The spectra functions supplied to `fmmesolve` must now be vectorized
+  (i.e. accept and return numpy arrays for frequencies and densities) and
+  must accept negative frequence (i.e. usually include a `w > 0` factor
+  so that the returned densities are zero for negative frequencies).
+- The number of sidebands to keep, `kmax` may only be supplied when using
+  the `FMESolver`
+- The `Tsteps` parameter has been removed from both `fsesolve` and
+  `fmmesolve`. The `precompute` option to `FloquetBasis` may be used
+  instead.
+
+Evolution of State Solver (essovle):
+
+- The function `essolve` has been removed. Use the `diag` integration
+  method with `sesolve` or `mesolve` instead.
+
+Steady-state solvers (steadystate module):
+
+- The `method` parameter and `solver` parameters have been separated. Previously
+  they were mixed together in the `method` parameter.
+- The previous options are now passed as parameters to the steady state
+  solver and mostly passed through to the underlying SciPy functions.
+- The logging and statistics have been removed.
+
+Correlation functions (correlation module):
+
+- A new `correlation_3op` function has been added. It supports `MESolver`
+  or `BRMESolver`.
+- The `correlation`, `correlation_4op`, and `correlation_ss` functions have been
+  removed.
+- Support for calculating correlation with `mcsolve` has been removed. 
+
+Propagators (propagator module):
+
+- A class interface, `qutip.Propagator`, has been added for propagators.
+- Propagation of time-dependent systems is now supported using `QobjEvo`.
+- The `unitary_mode` and `parallel` options have been removed.
+
+Correlation spectra (spectrum module):
+
+- The functions `spectrum_ss` and `spectrum_pi` have been removed and
+  are now internal functions.
+- The `use_pinv` parameter for `spectrum` has been removed and the
+  functionality merged into the `solver` parameter. Use `solver="pi"`
+  instead.
+
+QuTiP core
+----------
+
+There have been numerous other small changes to core QuTiP features:
+
+- A new function `qft` has been added that returns the quantum Fourier
```

Comment:
I think `qft` was moved from the `qutip.qip.algorithm` submodule?

Reply:
Thanks, Boxi. Corrected.